examples/webgl_gpgpu_birds.html

```diff
@@ -445,18 +371,19 @@
 				document.addEventListener( 'touchstart', onDocumentTouchStart, false );
 				document.addEventListener( 'touchmove', onDocumentTouchMove, false );
 
-				//
-
 				window.addEventListener( 'resize', onWindowResize, false );
 
 				var gui = new GUI();
 
-
 				var effectController = {
+
 					separation: 20.0,
 					alignment: 20.0,
 					cohesion: 20.0,
-					freedom: 0.75
+					freedom: 0.75,
+					size: 0.1,
```

Comment:
This is a personal preference - I think it looks even better with a slightly bigger size, around 0.2.

Reply:
Parrots are 0.2, flamingos 0.1 ;)